hue/hue.sh

```diff
@@ -56,20 +58,64 @@ EOF
 
   sed -i '/<\/configuration>/e cat hdfs-site-patch.xml' \
        /etc/hadoop/conf/hdfs-site.xml
-       
+
   cat >> /etc/hue/conf/hue.ini <<EOF
     # Defaults to $HADOOP_MR1_HOME or /usr/lib/hadoop-0.20-mapreduce
-    hadoop_mapred_home=/usr/lib/hadoop-mapreduce 
+    hadoop_mapred_home=/usr/lib/hadoop-mapreduce
 EOF
-       
-  # Replace localhost with hostname.
+
+  # Fix webhdfs_url
+  sed -i 's/## webhdfs_url\=http:\/\/localhost:50070/webhdfs_url\=http:\/\/'"$(hdfs getconf -confKey  dfs.namenode.http-address)"'/' /etc/hue/conf/hue.ini
```

Comment:
It's painful, but could you enforce line length (at most 100 prefreably 80 chars) breaking strings with \ and using ""

Reply:
Done